app/controllers/users_controller.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  before_action :find_user, only: %i[show update]
+  before_action :authorize_request, except: :create
```

Comment:
To improve readability you can use `skip_before_action :authorize_request, only: :create`

Reply:
This was still pending, you can push the changes we did today and work on this comment next.